rest/model/subscriber_test.go

```diff
@@ -106,8 +106,9 @@ func TestSubscriberModelsWebhook(t *testing.T) {
 	assert := assert.New(t)
 
 	target := event.WebhookSubscriber{
-		URL:    "foo",
-		Secret: []byte("bar"),
+		URL:     "foo",
+		Secret:  []byte("bar"),
+		Headers: []event.WebhookHeader{},
```

Comment:
is this necessary?

Reply:
I believe the test hit a nil pointer when this wasn't there